dataverse-webapp/src/main/webapp/resources/js/dv_rebind_bootstrap_ui.js

```diff
@@ -336,6 +336,133 @@ function reinitializePrimefacesComponentsJS() {
            }
         });
     }
+
+    if(PrimeFaces.widget.TextEditor) {
```

Comment:
Why we need this additional config? Shouldn't the editor work out-of-the-box?

Reply:
Precisely that - the out-of-the-box behavior is not wcag compatible so we needed to change it. That was the main goal of this task.